### DISCUSSION TITLE
您好,我在自己的项目中引入了您的工具包，但我复制了您在src/viewes/example/tools/analysis代码，为什么不显示呢

### DISCUSSION THREAD

**Discussion by sabkfd** [dengxiaoning/cesium_dev_kit#17] [Q&A] [answered]
您好,我在自己的项目中引入了您的工具包，但我复制了您在src/viewes/example/tools/analysis代码，为什么不显示呢
![image](https://github.com/dengxiaoning/cesium_dev_kit/assets/169125769/7294718f-1594-4fd5-a1ad-27784563d9b7)
![image](https://github.com/dengxiaoning/cesium_dev_kit/assets/169125769/a92b3df2-912c-425d-aa18-f567110c8584)
这是运行结果
![image](https://github.com/dengxiaoning/cesium_dev_kit/assets/169125769/4b9586a4-b146-4e28-aec3-f54ce02d2eb6)
恳请解答

**Comment by dengxiaoning** [comment]
你好，根据你的问题推测可能做以下几种检查：
1、F12检测一下浏览器控制台是否有异常显示
2、检查cesium_dev_kit依赖包是否已经正确安装
3、你可以直接git clone https://github.com/dengxiaoning/cesium_kit_test.git  这个[项目](https://github.com/dengxiaoning/cesium_kit_test)已经增加三维分析案例测试
![image](https://github.com/dengxiaoning/cesium_dev_kit/assets/26786620/0f380685-11c1-4dc5-ba47-4f2aad9da4b5)

**Reply by sabkfd** [reply]
您好，控制台没有显示异常，我按照您的下载了图片所示的项目，但是无法运行
![image](https://github.com/dengxiaoning/cesium_dev_kit/assets/169125769/45aa503b-c048-4ce4-943a-92372f27030a)
我在网上查了方法有让npm install的，但没有用，node_modules是空的
我之前的项目是用的vite,不知道缺了什么，
![image](https://github.com/dengxiaoning/cesium_dev_kit/assets/169125769/4fef8d1d-0fe4-47ee-a648-bc25f0cfccf6)

**Reply by dengxiaoning** [reply]
npm install  是否有异常提示？  node 建议使用  v16.0.0（可以使用gnvm 下载多个node 进行切换）

**Reply by sabkfd** [reply]
![image](https://github.com/dengxiaoning/cesium_dev_kit/assets/169125769/b52ea4c7-f58a-444a-9156-2669cba494b1)
确实有报错信息

**Reply by dengxiaoning** [reply]
npm 版本问题，建议使用  8.19.4 
![image](https://github.com/dengxiaoning/cesium_dev_kit/assets/26786620/56857ee8-3392-4351-9d35-d508fd3e8e9e)

**Reply by dengxiaoning** [reply]
npm set registry https://registry.npmjs.org/

**Reply by dengxiaoning** [reply]
npm config set registry https://registry.npmmirror.com

**Reply by sabkfd** [reply]
node已经降为16.0.0，npm也是8.19.4，镜像也设置了还是报错
![image](https://github.com/dengxiaoning/cesium_dev_kit/assets/169125769/d2781e45-0994-40cb-8c7d-5bd2ec2a09d0)

**Reply by dengxiaoning** [reply, marked as answer]
npm config list
npm config get proxy
npm config get https-proxy
npm config set https-proxy null
npm config set proxy null
npm config set registry https://registry.npmjs.org/

**Reply by sabkfd** [reply]
感谢指导，我已经加载出来了，但我的项目是1.116版本，而您的是1.105，还是很难融合，有什么好的方法吗

**Reply by dengxiaoning** [reply]
没关系，高于1.102 就行

**Reply by sabkfd** [reply]
非常感谢您的指导，我将自己的版本退到1.105版本就能加载了。
